base/task.jl

```diff
@@ -338,6 +364,28 @@ macro sync(block)
     end
 end
 
+"""
+    @syncany
+
+Wait until all lexically-enclosed uses of `@async`, `@spawn`, `@spawnat` and `@distributed`
+are complete, or at least one of them has errored. The first exception is immediately
+rethrown. It is the responsibility of the user to cancel any still-running operations
+during error handling.
+
+!!! Note
+    This interface is experimental and subject to change or removal without notice.
```

Comment:
:+1: for marking this experimental. Otherwise, implementing structured concurrency #33248 becomes much harder.